CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
 - [#630](https://github.com/oauth2-proxy/oauth2-proxy/pull/630) Add support for Gitlab project based authentication (@factorysh)
 - [#907](https://github.com/oauth2-proxy/oauth2-proxy/pull/907) Introduce alpha configuration option to enable testing of structured configuration (@JoelSpeed)
 - [#938](https://github.com/oauth2-proxy/oauth2-proxy/pull/938) Cleanup missed provider renaming refactor methods (@NickMeves)
+- [#928](https://github.com/oauth2-proxy/oauth2-proxy/pull/928) Allow * in the email-domain list. (@morarucostel)
```

Comment:
We should update this CHANGELOG to reflect the compromise functionality we ended up with that didn't have the security issues.